docker/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN go get -u golang.org/x/lint/golint
 
 # get and compile cadence-server
 RUN git clone https://github.com/uber/cadence.git /go/src/github.com/uber/cadence
-RUN cd /go/src/github.com/uber/cadence && git checkout $git_branch && make bins_nothrift
+RUN cd /go/src/github.com/uber/cadence && git fetch && git checkout $git_branch && make bins_nothrift
```

Comment:
`RUN git clone https://github.com/uber/cadence.git /go/src/github.com/uber/cadence`
should already pull the newest change?

Reply:
need fetch before checkout to branch other than master.